dockertest/dockercmd_unittests.py

```diff
@@ -284,13 +282,13 @@ def test_no_fail_docker_cmd(self):
 
         docker_command = self.dockercmd.DockerCmd(self.fake_subtest,
                                                   'unittest_fail')
-        self.assertRaises(self.xceptions.DockerExecError,
+        self.assertRaises(self.output.xceptions.DockerExecError,
```

Comment:
\*sob\*. That's exactly how I think it should work, and that's exactly what I've been struggling to make happen, and why my forehead hurts so much from banging on my desk. I am flummoxed!